packages/protocol/package.json

```diff
@@ -21,7 +21,8 @@
     "test": "forge test -vvv --gas-report --fuzz-seed $(date +%s) --nmp test/L1/TaikoL1.sim.sol",
     "test:coverage": "forge coverage --report lcov",
     "test:genesis": "./genesis/generate_genesis.test.sh",
-    "upgrade": "./script/upgrade_to.sh"
+    "upgrade": "./script/upgrade_to.sh",
+    "install:solc": "mkdir -p ./bin && curl -o ./bin/solc https://github.com/ethereum/solidity/releases/download/v0.8.18/solc-static-linux && chmod +x ./bin/solc"
```

Comment:
emmm actually we have a script here: https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/script/download_solc.sh